examples/helpers/cmake/gen_android_example.cmake.in

```diff
@@ -55,6 +55,12 @@ function(gen_android_example_application
         set(EDGE_SDK_EXAMPLE_NETWORK_PROTOCOL_JAR "-DEDGE_SDK_NETWORK_PROTOCOL_JAR=${EDGE_SDK_NETWORK_PROTOCOL_PATH}/EdgeNetworkProtocol.jar")
     endif()
 
+    set(BOOST_ROOT "full path to the boost library folder in your env")
+    # Try to find the path to the boost
+    if (DEFINED olp-cpp-sdk-core_BINARY_DIR)
+        set(BOOST_ROOT "$ENV{WORKSPACE}/build/external/boost/external_boost")
```

Comment:
```suggestion
        set(EDGE_SDK_BOOST_ROOT BOOST_ROOT)
```
Alternatively, we can omit declaring the `EDGE_SDK_BOOST_ROOT` variable and pass just the `BOOST_ROOT` to `app/build.gradle.in`. For me both approaches are OK.